src/erlcloud_aws.erl

```diff
@@ -146,7 +146,7 @@ aws_region_from_host(Host) ->
         [_, Value, _, _ | _Rest] ->
             Value;
         _ ->
-            "us-east-1"
+            application:get_env(erlcloud, aws_region, "us-east-1")
```

Comment:
stylish: default it in record define so to have only a single place for deafults